apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -20,8 +20,8 @@
     <div class="inline-flex items-center mt-4 mb-2">
       <SfRating size="xs" :value="productGetters.getAverageRating(reviewAverage)" :max="5" />
       <SfCounter class="ml-1" size="xs">{{ productGetters.getTotalReviews(reviewAverage) }}</SfCounter>
-      <SfLink href="#" variant="secondary" class="ml-2 text-xs text-neutral-500">
-        {{ $t('reviewsCount', { count: productGetters.getTotalReviews(reviewAverage) }) }}
+      <SfLink href="#customerReviewsAccordion" variant="secondary" class="ml-2 text-xs text-neutral-500">
+        {{ $t('showAllReviews') }}
```

Comment:
Why not using the smooth solution from the pwa?

`const scrollToReviews = () => {
      document
        .getElementById('customerReviewsAccordion')
        .scrollIntoView({ behavior: 'smooth', block: 'start', inline: 'nearest' });
    };`